### PR TITLE
Adds continuous integration setup with tox and coverage

### PR DESCRIPTION
**Pull request by lukpueh**
- Decorates not implemented unittests to be skipped
- Updates tox.ini to install requirements, run coverage and call `runtests.py` instead of `setup.py test`.
- Updates .travis.yml to use tox and coveralls. *Running coveralls independently of Travis CI would require to "provide a repo_token in .coveralls.yml"*

### CODE REVIEW

**Comment by lukpueh** [comment]
Let's make this a 100%!

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
 
 install:
     - pip install tox
+    - pip install coveralls
 
 script:
     tox
+
+after_success:
+    coveralls
```
Comment (SantiagoTorres) [inline .travis.yml]: it'd be nice if we had a newline here

File: test/test_toto_verify.py [hunk 2]
```diff
@@ -8,6 +8,8 @@
 
 class Test(unittest.TestCase):
 
+  @unittest.skip("Missing test implementation")
+
```
Comment (SantiagoTorres) [inline test/test_toto_verify.py]: can we remove this whitespace?

File: tox.ini [hunk 3]
```diff
@@ -5,15 +5,22 @@
...
 
 deps =
-   -rrequirements.txt
-   coveralls
+    -rrequirements.txt
+    pycrypto
+    cryptography
+    pytest
+    PyNaCl
+    coverage
```
Comment (SantiagoTorres) [inline tox.ini]: maybe we could create a file called requirements-ci.txt and add these there. Just keep -rrequirements-ci.txt...

File: tox.ini [hunk 4]
```diff
@@ -5,15 +5,22 @@
...
-
 
 [testenv]
-passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
-commands =
-    coverage run --source=toto setup.py test
-    coveralls
+setenv =
+    SIMPLE_SETTINGS=toto.settings
+    # PYTHONPATH = {toxinidir}
```
Comment (SantiagoTorres) [inline tox.ini]: Do we need this commented out?

**Review by SantiagoTorres — CHANGES_REQUESTED** [review]

**Review by SantiagoTorres — APPROVED** [review]

**Comment by SantiagoTorres** [comment]
@lukpueh I think this is ready to merge? 👍

**Comment by lukpueh** [comment]
Thanks for the review, @SantiagoTorres!

**Pull request merged**